docs/root/_include/cert_stats.rst

```diff
@@ -0,0 +1,5 @@
+.. csv-table::
+   :header: Name, Type, Description
+   :widths: 1, 1, 2
+
+   ``seconds_since_epoch_of_expiration``, Gauge, Number of seconds since UNIX epoch of the expiration date of the certificate.
```

Comment:
I think the name of the metric could be shorted to `expiration_unix_time`. I think "UNIX time" is widely-enough understood that this will be a more precise and concise name.

Reply:
Sounds good. Let's go with `expiration_unix_time`